website/content/v1.11/advanced/etcd-maintenance.md

```diff
@@ -74,3 +74,114 @@ NODE         MEMBER             DB SIZE   IN USE             LEADER
+Validate the downgrade target version before enabling the downgrade:
+
+- We only support downgrading one minor version at a time, e.g. downgrading from v3.6 to v3.4 isn't allowed.
+- Please do not move on to next step until the validation is successful.
+
+```bash
+$ talosctl -n <CP1> etcd downgrade validate 3.5
+NODE         MESSAGE
+172.20.0.2   downgrade validate success, cluster version 3.6
+```
+
+#### Step 4: enable downgrade
+
+```bash
+$ talosctl -n <CP1> etcd downgrade enable 3.5
+NODE         MESSAGE
+172.20.0.2   downgrade enable success, cluster version 3.6
+```
+
+After enabling downgrade, the cluster will start to operate with v3.5 protocol, which is the downgrade target version.
+In addition, etcd will automatically migrate the schema to the downgrade target version, which usually happens very fast.
+Confirm the storage version of all servers has been migrated to v3.5 by checking the endpoint status before moving on to the next step.
+
+```bash
+$ talosctl -n <CP1>,<CP2>,<CP3> etcd status
+NODE         MEMBER             DB SIZE   IN USE            LEADER             RAFT INDEX   RAFT TERM   RAFT APPLIED INDEX   LEARNER   PROTOCOL   STORAGE   ERRORS
+172.20.0.3   912415ee6ed360c4   3.5 MB    1.9 MB (54.92%)   a49c021e76e707db   5152         2           5152                 false     3.6.4      3.5.0
+172.20.0.2   a49c021e76e707db   3.5 MB    1.9 MB (54.64%)   a49c021e76e707db   5152         2           5152                 false     3.6.4      3.5.0
+172.20.0.4   a2b8a1f794bdd561   3.6 MB    1.9 MB (54.44%)   a49c021e76e707db   5152         2           5152                 false     3.6.4      3.5.0
+```
+
+> Note: Once downgrade is enabled, the cluster will remain operating with v3.5 protocol even if all the servers are still running the v3.6 binary, unless the downgrade is canceled with `talosctl -n <CP1> downgrade cancel`.
+
+#### Step 5: patch machine config of node
+
+Before patching the node, check if the etcd is leader.
+We recommend downgrading the leader last.
+If the server to be downgraded is the leader, you can avoid some downtime by `forfeit-leadership` to another server before stopping this server.
+
+```bash
+talosctl -n <CP1> etcd forfeit-leadership
+```
+
+Create a file with the patch pointing to desired etcd image:
+
+```yaml
+# etcd-patch.yaml
+cluster:
+  etcd:
+    image: gcr.io/etcd-development/etcd:v3.5.22
+```
+
+Apply patch to the machine with same configuration but with the new etcd version.
+
+```bash
+$ talosctl -n <CP1> patch machineconfig --patch @etcd-patch.yaml --mode reboot
+patched MachineConfigs.config.talos.dev/v1alpha1 at the node 172.20.0.2
+Applied configuration with a reboot
+```
+
+Verify that each member, and then the entire cluster, becomes healthy with the new v3.5 etcd:
+
+```bash
+$ talosctl -n <CP1>,<CP2>,<CP3> etcd status
+NODE         MEMBER             DB SIZE   IN USE            LEADER             RAFT INDEX   RAFT TERM   RAFT APPLIED INDEX   LEARNER   PROTOCOL   STORAGE   ERRORS
```

Comment:
Won't copying the output into the command make it much less copy-pasteable?

Reply:
Here is improved version:

<img width="768" height="137" alt="image" src="https://github.com/user-attachments/assets/904ae81c-d7c1-4e19-b42a-c86145a8039f" />